starter-code/iteration-2/main.ts

```diff
@@ -1,6 +1,39 @@
 // Add the reference to the "TodoInterface"
+import {TodoInterface} from './todoInterface'
+
 
 // 1. Create a class Todo that implements the Interface created before.
+class Todo implements TodoInterface {
+
+constructor(public testArray:Array<string>){}
+
+addTask(task){
+    
+    this.testArray.push(task);
+    var number= this.testArray.length;
+    return number;
+    console.log (task+"was added");
+}
+
+listAllTasks(task){
+
+
+for (var i=0;i<this.testArray.length;i++) {
```

Comment:
you could have used a forEach as well here: 
`listAllTasks(): void {
    this.todoList.forEach((task) => {
      console.log(task);
    });
  }`